utils/build/docker/golang/parametric/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.20.0
 	google.golang.org/grpc v1.57.1
 	google.golang.org/protobuf v1.30.0
-	gopkg.in/DataDog/dd-trace-go.v1 v1.39.0-alpha.1.0.20240419091955-7df262e804ad
+	gopkg.in/DataDog/dd-trace-go.v1 v1.39.0-alpha.1.0.20240424121809-d9b1b369853f
```

Comment:
Why is this using this oddly old `dd-trace-go` version?

Reply:
It's actually not, but that's how the version is displayed